src/App.tsx

```diff
@@ -21,46 +20,48 @@ const intialState = {
   },
   questions: [
     {
+      category: 'Here',
       compliance: [
         {
-          standard: 'string'
+          standard: ''
         }
       ],
-      description: 'description',
-      title: 'title',
+      description: '',
+      title: '',
       queries: [
         {
-          query: 'query',
+          query: '',
         }
       ],
-      tags: ['tag'],
+      tags: [''],
     }
   ]
 }
 
 function App() {
   const [managedQuestions, setManagedQuestions] = useState<ManagedQuestionJSON>(intialState)
   const [allTags, setAllTags] = useState<string[]>([])
+  const [allCategories, setAllCategories] = useState<string[]>([])
   const [search, setSearch] = useState('')
   const [themeDark, setTheme] = useState<boolean>(false)
 
-
-  console.log(search)
-
   useEffect(() => {
     fetchQuestions().then((r : ManagedQuestionJSON) => {
       const tags : string[] = r.questions.map((question : Question) => {
         return question.tags
       }).flat(2)
+      const categories : string[] = r.questions.map((question : Question) => {
+        return question.category
+      }).flat(1)
```

Comment:
Interesting. In the future, you should use a lodash method for flat map or use a reduce function.